airflow/utils/mixins.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
Filename should be less generic (we should move away from keeping many classes in one dir if they are not lightweight or are unrelated).